lmms_eval/models/qwen2_vl.py

```diff
@@ -0,0 +1,274 @@
+import os
+import uuid
+import warnings
+from typing import List, Optional, Tuple, Union
+
+import torch
+from accelerate import Accelerator, DistributedType
+from tqdm import tqdm
+
+from lmms_eval import utils
+from lmms_eval.api.instance import Instance
+from lmms_eval.api.model import lmms
+from lmms_eval.api.registry import register_model
+
+warnings.simplefilter("ignore", category=DeprecationWarning)
+warnings.filterwarnings("ignore")
+
+from loguru import logger as eval_logger
+from qwen_vl_utils import process_vision_info
```

Comment:
I think it is possible to not use qwen vl  utils to process images right? Otherwise the model may not be registered into the model unless we have to modify the dependency